wiki/deployment.md

```diff
@@ -0,0 +1,23 @@
+# Deployment
+
+DataMade hosts a [staging version](https://lametro.datamade.us/) and [production version](https://boardagendas.metro.net/) of Metro Councilmatic. 
+
+Both sites reside on the [`metro-councilmatic` server](https://github.com/datamade/server-metro-councilmatic), an EC2 instance spun up in late 2018, after [the original Councilmatic server ran out of diskspace](https://github.com/datamade/devops/issues/55). Metro developers should be able to shell into this server, e.g., [for executing management commands](commands.md). If you cannot, then talk to your friendly DataMade devops team.
+
+### Patterns and practices
+
+Metro deployments follow a typical DataMade pattern: merging or pushing to master deploys to the staging site, and pushing a tag deploys to the production site. Metro Councilmatic also uses the "classic" style of DataMade deployment - or, conversely, Metro does not follow the contempary ["zero downtime" deployment pattern](https://github.com/datamade/deploy-a-site/blob/master/Zero-Downtime-Deployment.md). As a result, the Metro site raises a 500 error for a couple minutes, during the course of a codedeploy lifetime.
```

Comment:
can we phrase this like, "_triggering_ metro deployments follows the typical datamade pattern... conversely, the metro deployment scripts do not use datamade's contemporary, zero-downtime pattern." helps to know which part of the deployment we're talking about, i think.

Reply:
nice - I like this